### DISCUSSION TITLE
Github pages produces a 404 error

### DISCUSSION THREAD

**Discussion by mullaney** [community/community#25327] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve set up a repo using vue and vue styleguidist. I’ve also set up an action that will build a new version of the styleguidist documentation every time there is a merge to master. The action appears to be working. It’s building the proper files and saving them to the gh_pages branch. Under settings, I have verified that gh_pages branch is the one I want to use and I get this message:</p>
<p>Your site is ready to be published at <a href="http://mullaney.github.io/benchprep-components/" rel="nofollow noopener">http://mullaney.github.io/benchprep-components/</a>.</p>
<p>However, when I go to that page I get a 404 error. Is there a step that I’m missing? It’s been over an hour since I last published it. And I have tried multiple browsers in case it’s a caching issue.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/1ecca3101d35fbdff54e54116a95f3b9bd629fc7.png" data-download-href="/uploads/short-url/4osKV5Od4sNZmvK3pfpQE46D7NB.png?dl=1" title="Screen Shot 2020-06-20 at 10.24.58 AM" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1998791/181090446-fcf583ff-99a5-4c60-9f6d-2f60620a6b2b.png" alt="Screen Shot 2020-06-20 at 10.24.58 AM" data-base62-sha1="4osKV5Od4sNZmvK3pfpQE46D7NB" width="610" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/1ecca3101d35fbdff54e54116a95f3b9bd629fc7_2_610x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/1/1ecca3101d35fbdff54e54116a95f3b9bd629fc7_2_915x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/1/1ecca3101d35fbdff54e54116a95f3b9bd629fc7.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/1ecca3101d35fbdff54e54116a95f3b9bd629fc7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-06-20 at 10.24.58 AM</span><span class="informations">936×767 65.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by mullaney** [comment]
<p>For reference here is the gh_pages branch:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d3711662044df9ed74a260facd2e0712fa37ecb8.png" data-download-href="/uploads/short-url/uav2aqZzzenlIxeriCmCD3HNioo.png?dl=1" title="Screen Shot 2020-06-20 at 10.25.40 AM" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1998791/181090448-32e6d450-9609-4e68-a125-5b3a115eba47.png" alt="Screen Shot 2020-06-20 at 10.25.40 AM" data-base62-sha1="uav2aqZzzenlIxeriCmCD3HNioo" width="690" height="369" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d3711662044df9ed74a260facd2e0712fa37ecb8_2_690x369.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/d3711662044df9ed74a260facd2e0712fa37ecb8_2_1035x553.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/d/d3711662044df9ed74a260facd2e0712fa37ecb8.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d3711662044df9ed74a260facd2e0712fa37ecb8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-06-20 at 10.25.40 AM</span><span class="informations">1263×677 87.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by mullaney** [comment, marked as answer]
<p>A message from github support:</p>
<p>Hi Kevin,</p>
<p>In order to be built and published for the very first time, a GitHub Pages repository must have a commit pushed to it by a user with admin permissions for the repository, such as the repository owner.</p>
<p>In this case, it looks like the only commits pushed to the  <code>gh-pages</code>  branch were from  <code>github-actions</code> .</p>
<p>Once a commit has been pushed to the repository by a user with admin permissions, you should find the site live and available shortly thereafter. Subsequent commits pushed by accounts with write access such as  <code>github-actions</code>  will then also trigger a build.</p>
<p>Sorry for the inconvenience; this issue is something that’s currently being looked at by our engineers as more and more Pages sites are using Actions.</p>
<p>I hope that helps to explain! Let us know if there’s anything else we can do.</p>
<p>Thanks,</p>
<p>Tammy<br>
GitHub Support</p>

**Comment by Yanjingzhu** [comment]
<p>I checked <a href="https://mullaney.github.io/benchprep-components/" rel="nofollow noopener">your page</a> , it could display now.<br>
Thank you for reporting this issue and share the response from GitHub Support, it would be helpful for other communities who meet the same issue.</p>
